stratum/hal/bin/nikss/main.cc

```diff
@@ -0,0 +1,65 @@
+#include "gflags/gflags.h"
+#include "stratum/glue/init_google.h"
+#include "stratum/glue/logging.h"
+#include "stratum/hal/lib/nikss/nikss_switch.h"
+#include "stratum/hal/lib/common/hal.h"
+#include "stratum/hal/lib/phal/phal_sim.h"
+#include "stratum/lib/security/auth_policy_checker.h"
+#include "stratum/lib/security/credentials_manager.h"
+
+extern "C" {
```

Comment:
I think that the `extern "C"` should be a part of nikss headers, see issue https://github.com/NIKSS-vSwitch/nikss/issues/90. I mean to add following code to these headers:
```c
#ifdef __cplusplus
extern "C" {
#endif
```
And a closing bracket in similar way.